src/test/python/testjson.py

```diff
@@ -158,6 +158,34 @@ def test_invalid_param_count_static(self):
         with self.assertRaises(TypeError):
             wdj.JsonArray.Parse(10, 20)
 
+    def test_JsonArray_GetMany(self):
+        a = wdj.JsonArray.Parse('[true, false, 42, null, [], {}, "plugh"]')
+        count, items = a.GetMany(1, 3)
+
+        self.assertEqual(count, 3)
+        self.assertEqual(len(items), 3)
+
+        self.assertFalse(items[0].GetBoolean())
+        self.assertEqual(items[1].GetNumber(), 42)
+        self.assertEqual(items[2].ValueType, 0)
+
+        # TODO: remove clear call after resolving leak issue
+        items.clear()
```

Comment:
As per the initial PR comment, this call is here to work around a crashing bug. I'm planning on reworking my ref counting code by moving to a smart pointer (a la https://github.com/zooba/pyptr) in a seperate PR